src/components/CallView/shared/ReactionToaster.vue

```diff
@@ -110,6 +120,11 @@ export default {
 		},
 
 		handleReaction(model, reaction) {
+			// prevent receiving anything rather than defined reactions in capabilities
+			if (!this.supportedReactions.includes(reaction)) {
+				return
+			}
+
```

Comment:
Do we need this check on frontend? Backend should handle it anyway.

Reply:
We don't have validation on backend, according to this: https://github.com/nextcloud/spreed/pull/9311#issue-1668282354
Still, any text could be sended with command: `document.querySelector('#localVideoContainer').__vue__.localCallParticipantModel.sendReaction('👎')`
So it seems fine to me to make an additional check here.